_includes/single-source/data-reliability-best-practices.md

```diff
@@ -0,0 +1,10 @@
+> 🚧 Your applications are only as reliable as the data that they use
+>
+> When you run a production application, be aware of the risks associated with each data provider that you use.
+
+To protect your applications and your users, apply the following guidelines to your application and oracle network design:
+
+- Be cautious when your applications use data providers with only a single data source, which can be a single point of failure. Any error or omission in the provider's data might negatively impact your application and its users.
+- Consider the incentives that malicious actors might have to manipulate the data from each data provider.
+- For nodes that provide pricing data, consider the liquidity of the exchange where that pricing data is coming from. Exchanges and asset pairs with low liquidity and high volatility are vulnerable to price manipulation. If your application relies on that pricing data, it is equally vulnerable to manipulation.
```

Comment:
@dwightjl this should be more about data providers and less about nodes.
```suggestion
For pricing data providers, consider the liquidity of the exchange where pricing data is coming from. Exchanges and asset pairs with low liquidity and high volatility are vulnerable to price manipulation. If your application relies on that pricing data, it is equally vulnerable to manipulation.
```

Reply:
Applied to the latest commit.